### PR TITLE
EOS-15156: Add check to remove only remote build url

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**
Signed-off-by: Anjali Somwanshi <anjali.somwanshi@seagate.com>

### CODE REVIEW

File: cli/src/cortx_prep.sh [hunk 1]
```diff
@@ -44,8 +44,11 @@ function install_prvsnr() {
         mount -t iso9660 /opt/isos/${cortx_iso} /tmp/iso_mount/cortx 2>&1 | tee -a ${LOG_FILE}
         mount -t iso9660 /opt/isos/${os_iso} /tmp/iso_mount/cortx-os 2>&1 | tee -a ${LOG_FILE}
 
+        echo "INFO: Backing up exisitng repositories" 2>&1 | tee -a ${LOG_FILE}
+        mv /etc/yum.repos.d /etc/yum.repos.d.bak
```
Comment (andkononykhin2) [inline cli/src/cortx_prep.sh:48]: @SomwanshiAnjali  It's not only backing up but also a cleanup. Why have we decided to clean existent repos here?

Comment (SomwanshiAnjali) [reply cli/src/cortx_prep.sh:48]: @andkononykhin2  
This change was asked by @ypise, Some systemd packages are getting installed as dependencies from the existent repos with unexpected versions hence we are removing them
@ypise  could you please explain more if I have missed anything

Comment (ypise) [reply cli/src/cortx_prep.sh:48]: @andkononykhin2 this is particularly for lab environment where we might get additional repos like redhat.repo (even on CentOS).
To ensure we have complete control during deployment and avoid any unexpected versions being installed, I proposed cleaning up the directory.

File: cli/src/cortx_prep.sh [hunk 2]
```diff
@@ -44,8 +44,11 @@ function install_prvsnr() {
         mount -t iso9660 /opt/isos/${cortx_iso} /tmp/iso_mount/cortx 2>&1 | tee -a ${LOG_FILE}
         mount -t iso9660 /opt/isos/${os_iso} /tmp/iso_mount/cortx-os 2>&1 | tee -a ${LOG_FILE}
 
+        echo "INFO: Backing up exisitng repositories" 2>&1 | tee -a ${LOG_FILE}
```
Comment (ypise) [inline cli/src/cortx_prep.sh:47]: Update this line to:
`echo "INFO: Backing up exisitng repositories" 2>&1 | tee -a "${LOG_FILE}"`

To fix Codacy error.

**Review by ypise — APPROVED** [review]

File: cli/src/cortx_prep.sh (hunk 2 above)
Comment (SomwanshiAnjali) [reply cli/src/cortx_prep.sh]: Fixed

**Pull request merged**